src/hooks/useKeyboardControls.ts

```diff
@@ -31,28 +30,10 @@ export const useKeyboardControls: ReactHook<UseKeyboardControlsProps, void> = ({
   totalColumns,
   isSeatTaken,
   takeSeat,
-  enterSelectedRoom,
 }) => {
   const { profile } = useUser();
   const { row, column } = profile?.data?.[venueId] ?? {};
 
-  /**
-   * enter
-   */
-  const enter = useCallback(() => {
-    if (!row || !column) return;
-
-    // TODO: implement: openRoomUrl(selectedRoom.url) ? can we do it in enterSelectedRoom?
-    enterSelectedRoom();
-  }, [row, column, enterSelectedRoom]);
-
-  useMousetrap({
```

Comment:
moved to RoomModal

Reply:
As per https://github.com/sparkletown/sparkle/pull/1108/files/36acbe1b56bfcb0eaeb2dac543748428e999a1be#r567550279, why was this moved out of here to `RoomModal`?